build.sbt

```diff
@@ -152,29 +152,6 @@ val testResourceSettings = {
   })
 }
 
-lazy val jsonConfig210 = crossProject(JVMPlatform)
```

Comment:
If we're removing this I believe we can also then remove `config/src/main/scala-2.10/*` right? Since we'll no longer be publishing `bloop-config` in 2.10.

Reply:
Yay! More removed code :D